sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -128,7 +128,11 @@ namespace Azure { namespace Core { namespace Http {
    */
   struct WinHttpTransportOptions final
   {
-    // Empty struct reserved for future options.
+    /**
+     * @brief When `true`, allows an invalid certificate authority. If this flag is set, the
+     * application does not receive a WINHTTP_CALLBACK_STATUS_FLAG_INVALID_CA callback.
+     */
+    bool IgnoreUnknownServerCert = false;
```

Comment:
Should we avoid abbreviations and use `IgnoreUnknownCertificateAuthority`?

nit: What should the reader infer from `server` here in the name?

Reply:
In a TLS 1.2 connection, there are two possible certificates: The server certificate and the client certificate. The server validates the client certificate, and the client validates the server certificate. This variable controls the validation of the server certificate, so it makes sense to me that the variable name should probably include "Server", since there is some level of ambiguity present.

Having said that, the variable doesn't control if it ignores an unknown server certificate, instead as you mentioned, the flag controls if an unknown CA should be allowed, which is somewhat different (a server certificate could be invalid if (for instance) the subject didn't match the IP address of the server - this flag doesn't disable that check).